src/SwedbankPay.Sdk/Common/OperationsBase.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace SwedbankPay.Sdk.Common
+{
+    public abstract class OperationsBase : IDictionary<LinkRelation, HttpOperation>
+    {
+        private readonly Dictionary<LinkRelation, HttpOperation> internalDictionary = new Dictionary<LinkRelation, HttpOperation>();
+
+        public ICollection<LinkRelation> Keys => internalDictionary.Keys;
+
+        public ICollection<HttpOperation> Values => internalDictionary.Values;
+
+        public int Count => internalDictionary.Count;
+
+        public bool IsReadOnly => false;
+
+        HttpOperation IDictionary<LinkRelation, HttpOperation>.this[LinkRelation rel] { get => internalDictionary.ContainsKey(rel) ? internalDictionary[rel] : null; set => internalDictionary[rel] = value; }
```

Comment:
Codacy found an issue: [Use string or an integral type here, or refactor this indexer into a method.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)